test/test_in_chroot.py

```diff
@@ -85,17 +85,17 @@ def _unpack_debootstrap_tarball(self, tarball, target):
     def test_normal_upgrade(self):
         print("Running normal unattended upgrade in chroot")
         options = MockOptions()
-        options.minimal_upgrade_steps = False
+        options.minimal_upgrade_steps = True
         # run it
         target = self._run_upgrade_test_in_real_chroot(options)
         # ensure we upgraded the expected packages
         self.assertTrue(
             self._verify_install_log_in_real_chroot(target, "ca-certificates"))
 
     def test_minimal_steps_upgrade(self):
```

Comment:
Maybe worth renaming this test as well - like test_not_minimal_steps or something?

Reply:
Ah, indeed!